File: Web/src/test/java/form/CookiesForm.java
package form;
import aquality.selenium.elements.interfaces.IButton;
import aquality.selenium.forms.Form;
import org.openqa.selenium.By;

public class CookiesForm extends Form {
    private final IButton buttonNo = getElementFactory().getButton(By.xpath("//button[contains(text(),'Not really, no')]"), "'Not really, no' button");

    public CookiesForm() {
        super(By.xpath("//p[contains(@class,'cookies__message')]"), "cookies form");
    }

    public void clickButtonNo() {
        buttonNo.click();
    }
}

File: Web/UserInterfaceTest/UserInterfaceTest/Forms/Pages/CookiesForm.cs
using OpenQA.Selenium;
using Aquality.Selenium.Elements.Interfaces;

namespace UserInterfaceTest.Forms.Pages
{
    public class CookiesForm : BaseAppForm
    {
        private IButton AcceptCookiesButton = ElementFactory.GetButton(By.XPath("//button[contains(@class,'transparent')]"),"Accept cookies button");
        
        public CookiesForm() : base(By.XPath("//p[@class='cookies__message']"), "Cookies message label")
        {
        }

        public void AcceptCookies() => AcceptCookiesButton.Click();
    }
}
